src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -212,7 +214,7 @@ class SourcesTree extends Component<Props, State> {
 
     if (!nodeHasChildren(item)) {
       const source = sources.get(item.contents.get("id"));
```

Comment:
I think we could set this a bit further above and reuse here.

Reply:
Done